Resources/Locale/en-US/kitchen/components/kitchen-spike-component.ftl

```diff
@@ -6,10 +6,10 @@ comp-kitchen-spike-deny-not-dead = { CAPITALIZE(THE($victim)) } can't be butcher
 comp-kitchen-spike-begin-hook-victim = { CAPITALIZE(THE($user)) } begins dragging you onto { THE($this) }!
 comp-kitchen-spike-begin-hook-self = You begin dragging yourself onto { THE($this) }!
 
-comp-kitchen-spike-kill = { CAPITALIZE(THE($user)) } has forced { THE($victim) } onto the spike, killing them instantly!
+comp-kitchen-spike-kill = { CAPITALIZE(THE($user)) } { CONJUGATE-HAVE($user) } forced { THE($victim) } onto { THE($this) }, killing { OBJECT($victim)} instantly!
 
-comp-kitchen-spike-suicide-other = { CAPITALIZE(THE($victim)) } has thrown themselves on a meat spike!
-comp-kitchen-spike-suicide-self = You throw yourself on a meat spike!
+comp-kitchen-spike-suicide-other = { CAPITALIZE(THE($victim)) } { CONJUGATE-HAVE($victim) } thrown { REFLEXIVE($victim) } on { THE($this) }!
```

Comment:
Should be "Victim threw REFLEXIVE on THIS"
We want "Myra threw herself on the meatspike"